corehq/apps/ota/views.py

```diff
@@ -113,13 +122,17 @@ def get_restore_params(request):
         'as_user': request.GET.get('as'),
         'has_data_cleanup_privelege': has_privilege(request, privileges.DATA_CLEANUP),
         'overwrite_cache': request.GET.get('overwrite_cache') == 'true',
+        'openrosa_version': openrosa_version,
     }
 
 
 def get_restore_response(domain, couch_user, app_id=None, since=None, version='1.0',
                          state=None, items=False, force_cache=False,
                          cache_timeout=None, overwrite_cache=False,
-                         force_restore_mode=None, as_user=None, has_data_cleanup_privelege=False):
+                         force_restore_mode=None,
+                         as_user=None,
+                         has_data_cleanup_privelege=False,
+                         openrosa_version=settings.OPENROSA_VERSION):
```

Comment:
Yeah, to me too... Should I just add it as a constant and set the default to be "1.0"?